default/scripting/monster_designs/ShipDesignManifest.focs.txt

```diff
@@ -0,0 +1,36 @@
+ShipDesignManifest
```

Comment:
Correct.

However, the copies in the resource directory provide the order for the default designs and the monsters.  The copy in the user data directory provides the order for the user's saved designs.